marker/processors/ignoretext.py

```diff
@@ -13,22 +13,34 @@
 
 class IgnoreTextProcessor(BaseProcessor):
     """
-    A processor for ignoring text blocks that are common elements in the document.
-
-    Attributes:
-        common_element_threshold (float):
-            The minimum fraction of pages that a block must appear in to be considered a common element.
-            Default is 0.6.
+    A processor for identifying and ignoring common text blocks in a document. 
+    These blocks often represent repetitive or non-essential elements, such as headers, footers, or page numbers.
     """
     block_types = (
-        BlockTypes.Text, BlockTypes.PageHeader, 
+        BlockTypes.Text, BlockTypes.PageHeader,
         BlockTypes.PageFooter, BlockTypes.SectionHeader,
         BlockTypes.TextInlineMath
     )
-    common_element_threshold = .20
-    common_element_min_blocks = 3
-    max_streak = 3 # The maximum number of blocks in a row to consider a common element
-    text_match_threshold = 90
+    common_element_threshold: Annotated[
+        float,
+        "The minimum ratio of pages a text block must appear on to be considered a common element.",
+        "Blocks that meet or exceed this threshold are marked as common elements.",
+    ] = 0.6
```

Comment:
This value changed

Reply:
went through the rest of the default values and audited those